xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/WCAGContext.java

```diff
@@ -192,14 +206,14 @@ private WCAGTestResults(String testMethodName, String url, String pageClassName,
                 .stream()
                 .filter(rule -> FAILS_ON_RULE.containsKey(rule.getId()) && !FAILS_ON_RULE.get(rule.getId()))
                 .collect(Collectors.toList());
-            this.warnCount = numberOfViolations(warningViolations);
+            this.warnCount = amountOfChecks(warningViolations);
             if (this.warnCount != 0) {
                 this.warnReport = AbstractXWikiCustomAxeReporter.getReadableAxeResults(testMethodName, pageClassName,
                     url, warningViolations);
             }
         }
 
-        private int numberOfViolations(List<Rule> violations)
+        private int amountOfChecks(List<Rule> violations)
```

Comment:
```suggestion
        private int numberOfChecks(List<Rule> violations)
```

Reply:
:+1: 

Sorry, I missremembered your comment [on this last PR ](https://github.com/xwiki/xwiki-platform/pull/2116#discussion_r1135503980) and regressed on the vocabulary used TT

Addressed in https://github.com/xwiki/xwiki-platform/pull/2299/commits/4c5d999ad68b8a55ec56f1db6dbe44a32c55b2e9